nix/craneBuild.nix

```diff
@@ -279,5 +279,23 @@ craneLib.overrideScope' (self: prev: {
       devimint = { };
     };
   };
+
+  wasm-tests = craneLib.mkCargoDerivation (self.commonArgs // {
+    pname = "fedimint-wasm-tests";
+    CARGO_BUILD_TARGET = "wasm32-unknown-unknown";
+    cargoToml = self.commonArgs.src + "/fedimint-wasm-tests/Cargo.toml";
+    cargoLock = self.commonArgs.src + "/fedimint-wasm-tests/Cargo.lock";
+    buildPhaseCargoCommand = "cargo build --tests --manifest-path fedimint-wasm-tests/Cargo.toml";
+    nativeBuildInputs = self.commonArgs.nativeBuildInputs ++ [ pkgs.breakpointHook ];
+    cargoArtifacts = null;
+    doInstallCargoArtifacts = false;
+    installPhaseCommand = ''
+      mv fedimint-wasm-tests/target/wasm32-unknown-unknown/debug/deps/fedimint*.wasm $out
+    '';
+    preBuild = ''
```

Comment:
There's a variable in `nix/toolchain.nix` that already exports env vars for wasm.